components/common/card/Card.module.css

```diff
@@ -1,22 +1,19 @@
 .card {
   align-items: flex-start;
```

Comment:
Please not `align-items: flex-start;` only comes into effect when work with `flex`.